micropython/urllib.urequest/urllib/urequest.py

```diff
@@ -12,7 +12,7 @@ def urlopen(url, data=None, method="GET"):
     if proto == "http:":
         port = 80
     elif proto == "https:":
-        import ussl
+        import ssl
```

Comment:
I suggest using `import tls` directly.  Because these libraries don't need to be backwards compatible.